docs/plugins/channel-safely.rst

```diff
@@ -52,18 +52,17 @@ Commands
 --------
 
 :runonce:           Run the safety procedures once to set the marker mode of designations.
-:rebuild:           Rebuild the designation group data. Intended for to be used in the event
-                    the marker mode isn't being set correctly (mostly for debugging).
+:rebuild:           Rebuild the designation group data. You should also read Troubleshooting.
 
 Features
 --------
 
 :require-vision:    Toggle whether the dwarves need vision of a tile before channeling to it can be deemed unsafe. (default: enabled)
-:monitor:           Toggle whether to monitor the conditions of active digs. (default: disabled)
+:risk-averse:       Toggles whether to use cave-in prevention. Designations are activated in stages
+                    and their priorities along edges are modified. (default: enabled)
+:monitoring:        Toggle whether to monitor the conditions of active digs. (default: disabled)
```

Comment:
The description of "monitoring" needs a bit more explanation, I think. How does monitoring change the behavior of the plugin from the player's perspective? Why would a player choose to enable it? If all it does is enable the `resurrect` option to work, maybe we can just get rid of it as an independent option and enable monitoring when `resurrect` is enabled?

Reply:
It cancels active digs (ie. has a worker), based on safety. Safety being the possibility of a dwarf falling, or directly causing a cave-in. Although not all cave-ins will be prevented, some are... basicalllly.. considered intentional. If an active dig, along with a second active dig, will cause a tile to fall.. then one of the jobs is postponed. (The one that started second) This canceled job is a feature of monitoring.

It may very well be **[*effectively*]** non-functional at this point. It is unclear whether this is the case.